### ISSUE TITLE
[Bug Report] Bot keeps skipping over items in a playlist, but can play them when requested individually.

### ISSUE THREAD

**Issue by vastofficial** [jagrosh/MusicBot#532]
**Describe the bug**
When requesting the bot to play a YouTube playlist certain items cannot be played within it. But when requesting the songs to be played as a single song, it works.

**To Reproduce**
Do a command !play playlist xyz
Do a command !play Ritt Momney Put You Records On

**Expected behavior**
Can play all items of the playlist.

**Debug Info**
```
System Properties:
  java.version = 1.8.0_252
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_252-b09
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.3.1
  Owner = 141302663022116864
  Prefix = 1
  AltPrefix = null
  MaxSeconds = 0
  NPImages = false
  SongInStatus = true
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.2.0_212
  JDA-Utilities Version = 3.0.4
  Lavaplayer Version = 1.3.59

Runtime Information:
  Total Memory = 38
  Used Memory = 22

Discord Information:
  ID = 658610915461169162
  Guilds = 1
  Users = 2
```

**Error message**
[09:03:15] [ERROR] [LocalAudioTrackExecutor]: Error in playback of hsm4poTWjMs
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException: null
[09:03:16] [ERROR] [LocalAudioTrackExecutor]: Error in playback of jIoEaTN7GGo
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:117)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException: null

**Comment by MichailiK** [comment]
Duplicate of #531

**Comment by vastofficial** [comment]
> Duplicate of or likely related to #531

Symptoms seem to differ quite a lot tho, the bot doesn't leave the channel.

**Comment by onlyyouX2** [comment]
The same error version = 0.3.1 .

**Comment by jagrosh** [comment]
This should be fixed as of 0.3.2
